DocumentsFromSnapshotMigration/src/test/java/org/opensearch/migrations/bulkload/ParallelDocumentMigrationsTest.java

```diff
@@ -39,24 +43,19 @@ public class ParallelDocumentMigrationsTest extends SourceTestBase {
     static final List<SearchClusterContainer.ContainerVersion> TARGET_IMAGES = List.of(SearchClusterContainer.OS_V2_14_0);
 
     public static Stream<Arguments> makeDocumentMigrationArgs() {
-        List<Object[]> sourceImageArgs = SOURCE_IMAGES.stream()
```

Comment:
I ended up not using the preloaded images, uploading all of the data it was only taking ~4 seconds.  While we will be repeating this cost for each test, it seems like this is nearly the same runtime when compared against the previous run.

I've left all the existing code in place so we are ready to jump to much larger data volumes in future test cases.